packages/emitter-framework/src/python/components/init-declaration.tsx

```diff
@@ -0,0 +1,53 @@
+import { children, Children, Declaration, DeclarationProps, mapJoin, Scope } from "@alloy-js/core";
+import { ParameterDescriptor } from "./parameter.jsx";
+import { usePythonNamePolicy } from "../name-policy.js";
+
+export interface InitDeclarationProps extends Omit<DeclarationProps, "name"> {
+  parameters?: Record<string, Children | ParameterDescriptor>;
+}
+
+function isParameterDescriptor(value: Children | ParameterDescriptor): value is ParameterDescriptor {
+  return (
+    typeof value === "object" && value !== null && Object.hasOwn(value, "type")
+  );
+}
+
+export function InitDeclaration(props: InitDeclarationProps) {
```

Comment:
Should this be ClassInitDeclaration to differentiate from other usages of init?

Reply:
Well, there's no other kind of init besides an init file in Python.